ladle/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": [
+    "plugin:prettier/recommended", 
```

Comment:
```suggestion
```
I don't like `eslint-plugin-pretter` since he make a lot of noise, anyway prettier check action is enabled in ci